python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -119,7 +119,8 @@ def compile(model: nn.Module,
                 scheduler: _LRScheduler = None,
                 metrics: List[Metric] = None,
                 onnx: bool = False,
-                quantize: bool = True):
+                quantize: bool = True,
```

Comment:
why are we making `quantize` to true by default? @TheaperDeng

Reply:
At that time, just to make sure quantized model can be loaded with
```python
model = Trainer.compile(model)
model.load_quantized_state_dict(torch.load(qfile))
```
And it shouldn't make too much of a difference if we ask users to explicitly set `quantize=True`
```python
model = Trainer.compile(model, quantize=True)
model.load_quantized_state_dict(torch.load(qfile))
```
We can change it to `False` by default.

Reply:
I think we should change it to False; we can add proper checking for all unbinded methods (such as load_quantized_state_dict) later.